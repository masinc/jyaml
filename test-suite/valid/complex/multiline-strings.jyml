# Multi-line strings with different chomping
"literal": |
  Line 1
  Line 2
  Line 3

"folded": >
  This is a long
  paragraph that will
  be folded into a
  single line.

"literal_strip": |-
  No trailing newline
  here

"folded_keep": >+
  Keep trailing
  newlines
