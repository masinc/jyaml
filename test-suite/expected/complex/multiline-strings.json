{"literal": "Line 1\nLine 2\nLine 3\n", "folded": "This is a long paragraph that will be folded into a single line.\n", "literal_strip": "No trailing newline\nhere", "folded_keep": "Keep trailing\nnewlines\n\n"}